packages/core/src/local-admin-api.ts

```diff
@@ -32,7 +38,36 @@ export class LocalAdminApi implements AdminApi {
 
   async startIndexingModelData(modelData: Array<ModelData>): Promise<void> {
     await this.indexApi.indexModels(modelData)
-    await this.syncApi.startModelSync(modelData.map((idx) => idx.streamID.toString()))
+
+    if (this.syncApi.enabled) {
```

Comment:
early return if syncApi is disabled will let you remove an indentation layer and make this a bit more readable I think